File: GetService/src/main/java/com/shukary/GetService/models/Employer.java
package com.shukary.GetService.models;

import javax.persistence.*;
import java.util.List;

@Entity
@Table(name = "employer")
public class Employer {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "employer_id")
    private Long employerId;

    private User user;

    private List<PendingContract> pendingContracts;

    private List<CurrentContract> currentContracts;

    private List<FinishedContract> finishedContracts;

}

File: GetService/src/webapp/GetService/src/app/models/employer.ts
import { Contract } from './contract.model';
import { CurrentContract } from './current-contract';
import { FinishedContract } from './finished-contract';
import { PendingContract } from './pending-contract';
import { User } from './user';

export class Employer{
    id: number;
    user: User;
    pendingContracts: PendingContract[];
    currentContracts: CurrentContract[];
    finishedContracts: FinishedContract[];
    contract: Contract;
}